### ISSUE TITLE
Amanda Day 23

### ISSUE THREAD

**Issue by AmandaLemmons** [tiy-hou-q2-2015-rails/day-23#2]
https://github.com/AmandaLemmons/day_22

I wasn't able to get the published_on to work :(

**Comment by joeypoon** [comment]
Hey Amanda, your migration 20150603214119_add_published_on_to_blogs.rb is empty so published_on doesn't exist in your blogs table.

**Comment by AmandaLemmons** [comment]
This is the part where I am confused. What exactly do I need to do in order for it not to be empty? Do I add changes in the migration of 20150603214119 in order for it to pop up in the schema?

**Comment by AmandaLemmons** [comment]
Is this where we add the column like we did in the challenge this morning?

**Comment by joeypoon** [comment]
Yeah, just do the add_column in the migration. Basically, the schema shows you what the database looks like. You can't change the schema directly, the only way to change the schema is by creating a migration. rails g migration doesn't do anything on its own, you'll still need to write what you want the migration to do. Then when you run rake db:migrate, the migration files will run and make the changes you specified.

**Comment by AmandaLemmons** [comment]
Well, I added the add_column in the migration and ran rake db:migrate and it didnt' show up.. I think I will call it a night and see if I have better luck in the morning. 

Thanks for your help Joey!

https://github.com/AmandaLemmons/day_22

**Comment by joeypoon** [comment]
My guess would be that rake db:migrate was already run before so the app thinks that the migration was already run. You would probably need to create a new migration or do the rake db:migrate:down VERSION=######## and then migrate again. You can check rake db:migrate:status to see if its status is "up". In the morning, anyways. Good night :).

**Comment by jwo** [comment]
Awesome help Joey!
- jesse
